src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -220,6 +220,10 @@ private void StartImportingBasicBlock(BasicBlock basicBlock)
                             // type with a runtime lookup
                             _dependencies.Add(GetGenericLookupHelper(ReadyToRunHelperId.TypeHandleForCasting, catchType), "EH filter");
                         }
+                        else
+                        {
+                            _dependencies.Add(_compilation.ComputeConstantLookup(ReadyToRunHelperId.TypeHandleForCasting, catchType), "EH");
```

Comment:
> "EH" vs. "EH filter"

Make the reason the same for both constant lookup and runtime lookup?

Also, the comment can be generalized and moved before `if (catchType.IsRuntimeDeterminedSubtype)`